features/support/generic_helpers.rb

```diff
@@ -8,10 +18,6 @@ def wait_for
   end
 end
 
-def wait_for_ajax
-  wait_for { page.evaluate_script('jQuery.active').zero? }
-end
-
 def wait_for_document_ready
   wait_for { page.evaluate_script('document.readyState').eql? 'complete' }
```

Comment:
This seems a little odd.  Capybara should wait for things to become available anyway without having to do this.  I always see integration tests as an automated user test - and we would not expect a user to run javascript before the page can be interacted with ?